tools/sync_up.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+
+const SWG_ROOT = path.join(__dirname, '..', 'node_modules', 'security-wg', 'vuln');
+const NPM_TARGET_ROOT = path.join(__dirname, '..', 'ecosystem');
+const CORE_TARGET_ROOT = path.join(__dirname, '..', 'core');
+
+// collect and set up ecosystem reports
+fs.readdirSync(path.join(SWG_ROOT, 'npm'))
```

Comment:
Maybe store this `join()` result in a variable since it's also part of the `join()` below.